src/test/java/uk/gov/companieshouse/web/emergencyauthcodeweb/transformer/company/impl/CompanyTransformerImplTest.java

```diff
@@ -60,5 +64,8 @@ void getCompanyDetailAllPopulated() {
 
         assertEquals(COMPANY_NAME, companyDetailReturned.getCompanyName());
         assertEquals(COMPANY_NUMBER, companyDetailReturned.getCompanyNumber());
+        assertEquals(StringUtils.capitalize(COMPANY_STATUS), companyDetailReturned.getCompanyStatus());
```

Comment:
Not sure if I should transform the constants here or have them in the right format beforehand?